test/plugins/plugins.test.js

```diff
@@ -35,9 +35,9 @@ test('Export plugins based on steps config', async (t) => {
       cwd,
       logger: t.context.logger,
       options: {
-        verifyConditions: ['./test/fixtures/plugin-noop', {path: './test/fixtures/plugin-noop'}],
-        generateNotes: './test/fixtures/plugin-noop',
-        analyzeCommits: {path: './test/fixtures/plugin-noop'},
+        verifyConditions: ['./test/fixtures/plugin-noop.cjs', {path: './test/fixtures/plugin-noop.cjs'}],
```

Comment:
we should probably make sure to mention in the release notes the need to include the extension when referencing plugin files since semantic-release will now execute in an ESM context. 

this isnt captured in a breaking change note in the commits, so we may need to modify the generated release notes to account for this

Reply:
i did end up capturing this in [`1e424a0` (#2569)](https://github.com/semantic-release/semantic-release/pull/2569/commits/1e424a01551900fac8b242c0da80f3571a75e24d) in case we happen to merge this without squashing, but i also started a list in the PR description to capture breaking changes assuming that we do squash